src/__tests__/seed.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS sort_key_cache
```

Comment:
Isn't his a duplication?

Reply:
yeah, this was suppose to be for tests, but it doen't make any sense to keep it.
Removed